.travis/publish_docs.sh

```diff
@@ -19,9 +21,20 @@ BWHITE='\e[1;37m'       # Bold White
 
 function display_usage() {
   printf "${GREEN}Usage:\n"
-  printf "${YELLOW}      ./prepare_site.sh [version|dev]\n\n"
-  printf "${GREEN}   Example: ${YELLOW}./prepare_site.sh 2.2 ${GREEN}- Prepare docs for 2.2 release.\n"
-  printf "${GREEN}   Example: ${YELLOW}./prepare_site.sh dev ${GREEN}- Prepare docs for current SNAPSHOT.\n$RESET"
+  printf "${YELLOW}      ./publish_docs.sh [version|dev]\n\n"
+  printf "${GREEN}   Example: ${YELLOW}./publish_docs.sh 2.2 ${GREEN}- Prepare docs for 2.2 release.\n"
```

Comment:
it is just used as an example and currently I don't think this file is filtered by maven which makes it harder to inject the latest value. 

with that said we could change this so that we don't use any specific version in the usage printout, or we could use some shell magic to grab the latest from the pom